### DISCUSSION TITLE
Training and Testing Datasets

### DISCUSSION THREAD

**Discussion by ghost** [weecology/DeepForest#448] [Q&A]
Hi! 
First of all, I would like to thank you for sharing your work with us. My question is about the dataset for training and testing the pre-trained model in ImageNet, the produced released NEON.pt. Is it all available at https://zenodo.org/record/5914554#.ZGOPrdLMKTe, or did you not provide it?

**Comment by ethanwhite** [comment]
I think we may have answered this one over email, but just in case we didn't then yes, everything you need should be at that link. If you can't find something you need just let us know.
